# RTL-proteins-AND-SGs
Open Access data for manuscript entitled Endogenous retrovirus-like proteins recruit UBQLN2 to stress granules and shape their functional biology

README: Endogenous retrovirus-like proteins recruit UBQLN2 to stress granules and shape their functional biology
Journal: Science
Manuscript: adu6354

 Milaganur Mohan, Harihar1;Fernandez, Martin2; Huang, Camellia1;  Lin, Rita1; Ryou, Jaimie1; Seyfried, Donald1; Grotewold, Nikolas1; Barget, Anna J2; Whiteley, Alexandra3; Barmada, Sami1; Basrur, Venkatesha1; Mosalaganti, Shyamal1; Paulson, Henry1; Sharkey, Lisa1;
Author affiliations

University of Michigan Medical School
University of Michigan
University of Colorado Boulder

Funding:
Office of Environmental Management, P30AG072931
National Institute of Health, T32GM141840
National Institute of Health, T32GM007863
National Institute of Health, R01NS097542
National Institute of Health, R01NS113943
National Institute of Health, 1R56NS128110-01
National Institute of Health, DP2GM150019-01
National Institute of Health, R35NS122302
National Institute of Health, NIH S10OD030275


Description of the data and file structure

The human genome is replete with sequences derived from foreign elements including endogenous retrovirus-like proteins of unknown function. Here we show that UBQLN2, a ubiquitin-proteasome shuttle factor implicated in neurodegenerative diseases, is regulated by the linked actions of two retrovirus-like proteins, RTL8 and PEG10. RTL8 confers on UBQLN2 the ability to complex with and regulate PEG10. PEG10, a core component of stress granules, drives the recruitment of UBQLN2 to stress granules under various stress conditions, but can only do so when RTL8 is present. Changes in UBQLN2, RTL8 or PEG10 levels further remodel the kinetics of stress granule disassembly and translation recovery. PEG10 also alters overall stress granule composition by incorporating select extracellular vesicle proteins. Within stress granules, PEG10 forms virus-like particles, underscoring the structural heterogeneity of this class of biomolecular condensates. Together, these results reveal an unexpected link between pathways of cellular proteostasis and endogenous retrovirus-like proteins.

Files and variables

File: Dryad_submission.zip
Description: Data are grouped in appropriately named folders corresponding to each figure.

Brief descriptions of the contents of each folder are as follows:

Figure 1: Files for volcano plot generation (from the Excel file titled “PRF_F_2022_H_PAUL_836_54856_54870_1_12_RF1_RF2_Only_Unique_Pro”, columns C (gene name), N (abundance ratio), and Q (p-value) were used in the R code), MEGA11 files containing alignment of RTL proteins for phylogenetic tree construction, representative images in .jpg or .tif format and quantification of Western blots measuring PEG10 levels in HeLa, HEK293, and SK-N-SH cells treated with siRTL8 and siPEG10, representative images and quantification of Western blots measuring PEG10 levels in RTL8 KO HeLa cells rescued with HA-RTL8C, HA-RTL8C DN, Myc-LDOC1 or FLAG-UBQLN2, and representative images and quantification of Western blots measuring PEG10 levels in WT, non-targeting control, and RTL8 KO HEK293 cells.

Figure 2: Representative images in .jpg or .tif format and quantification of Western blots showing UBQLN2-PEG10 co-IP after UBQLN2 IP with and without crosslinking, representative images and quantification of Western blots showing UBQLN2-PEG10 co-IP after UBQLN2 IP in RTL8 KO HeLa cells rescued with HA-RTL8C full-length or DN, or PEG10 KO HeLa cells rescued with V5-PEG10 gag-pol full-length or DPPR, and representative PLA images in lif and jpg format, contrast and brightness settings file for LAS X, CellProfiler and Prism analyses files for UBQLN2-PEG10 PLA experiments on non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells.

Figure 3: Representative Western blot images in .jpg or .tif format of biotinylated isoxazole precipitation studies, .xls or .csv files showing overlapping cluster counts of dSTORM images across 3 replicates, analysis parameters on CODI (https://alto.codi.bio), representative dSTORM images in jpg format, and Prism file of analysis, and representative PLA images in lif and tiff formats, contrast and brightness settings file for LAS X, CellProfiler and Prism analyses files for UBQLN2-PEG10 PLA experiments on wild-type HeLa cells stressed with sodium arsenite or heat-shocked.

Figure 4: Representative images in .lif and .tif formats, contrast and brightness settings file for LAS X, CellProfiler and Prism analyses files for immunofluorescence staining of arsenite-treated non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells, arsenite treated RTL8 KO HeLa cells rescued with HA-RTL8C full-length or DN, U2OS cells transfected with V5-PEG10 gag, gag-pol full-length or gag-pol DPPR. 

Figure 5: Representative images in .lif and .jpg formats, representative videos in .mp4 format, contrast and brightness settings file for LAS X, Prism and .xlsx analyses files for live-cell imaging experiments capturing stress granule assembly and disassembly, representative images of Western blots in jpg format of translation recovery in arsenite-treated non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells.

Figure 6: Representative images in .lif and .jpg formats, contrast and brightness settings file for LAS X for immunofluorescence staining of ATXN10 in arsenite-treated non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells, Files for volcano plot, heat map generation, and Venn diagram generation (from the Excel file titled “Proteins only S-N ratio Unique-Razor”, columns A (gene name), G-I (abundance ratios), and J-L (p-values) were used in the R code) for stress granule IP analysis. The folder also contains .xlsx and .csv files which list extracellular vesicle proteins (from Xu et al., Journal of Clinical Investigation, 2016), stress granule proteins split into tiers (from RNA granule database: Millar et al., 2023, Chemical Reviews), and PEG10 interactors from the BioGrid database.

Figure 7: Adobe Illustrator files with representative images of cryo-CLEM data. Movies in .mp4 format of three representative tomograms showing PEG10 VLPs in stress granules.

Supp Figure 1: Representative images in .jpg or .tif format of Western blots in HEK293 cells stably transfected with HA-RTL8C; RIPA lysates of non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells; RIPA lysates of non-targeting WT, non-targeting control, and RTL8 KO HEK293 cells for UBQLN2 clients and PEG10 regulating factors, or human CNS lines treated with siRTL8 or siPEG10.

Supp Figure 2: .jpg file of quantification of PEG10 gag levels in Figure 1G, .tiff file of sequence alignments, structural alignment files in .json and .cif formats (analyzed on PyMoL v3.0.0), representative images in .sgd (analyzed using GeneSys v1.6.1.0 and GeneTools v4.3.8.0), .jpg or .tif format and quantification of Western blots measuring PEG10 levels in UBQLN TKO HEK293 cells rescued with HA-RTL8C and FLAG-UBQLN2 or UBQLN TKO HEK293 cells treated with siRTL8 and rescued with siRNA-resistant HA-RTL8C.

Supp Figure 3: Representative images in .jpg or .tif format of Western blots showing UBQLN2-PEG10 co-IP after PEG10 IP in RTL8 KO HeLa cells rescued with HA-RTL8C full-length or DN, or PEG10 KO HeLa cells rescued with V5-PEG10 gag-pol full-length or DPPR. Representative images in .jpg or .tif format and quantification of Western blots comparing RTL8 pulldown after UBQLN2 IP in PEG10 KO cells and PEG10 IP in UBQLN2 KO cells.

Supp Figure 4: Representative images in .lif and .tif formats, contrast and brightness settings file for LAS X of immunofluorescence staining in unstressed non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells or undergoing heat shock, hyperosmotic stress, or double-stranded RNA stress. Representative images in .lif and .tif formats, contrast and brightness settings file for LAS X of immunofluorescence staining of wild-type HeLa cells treated with siRTL8, siPEG10, or siUBQLN2, subjected to arsenite stress.

Supp Figure 5: Representative images in .lif and .tif formats, contrast and brightness settings file for LAS X, CellProfiler and Prism analyses files of immunofluorescence staining for HA-RTL8C transfected into arsenite-stressed non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells. Representative images in .lif, .jpg, and .tif formats, contrast and brightness settings file for LAS X of immunofluorescence staining in arsenite-stressed non-targeting control and RTL8 KO HEK293 cells or non-targeting control, PEG10 KO, and PEG10 gag-pol KO HeLa cells, and representative Western blot images of RIPA lysates from non-targeting control, PEG10 KO, and PEG10 gag-pol KO HeLa cells in .jpg format.

Supp Figure 6: Representative Western blot images of RIPA lysates from unstressed non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells expressing endogenous G3BP1 fused to mScarlet in .jpg format. Prism analysis file of stress granule area and G3BP1-mScarlet intensity after 20 minutes of arsenite treatment in the aforementioned lines.

Supp Figure 7: Representative Western blot images in .jpg and .sgd format and quantification (in .prism and .xlsx format) of RIPA lysates from non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells probed for ATXN10 and TCAF1. Heatmaps and volcano plots of stress granule IP analyses in .jpg format. Representative images in .lif and .jpg formats, contrast and brightness settings file for LAS X of immunofluorescence staining for ATXN10 in arsenite-stressed non-targeting control, RTL8 KO, PEG10 KO, and UBQLN2 KO HeLa cells. Representative Western blot images of stress granule IP samples from unstressed and stressed HeLa cells in .jpg format.

Supp Figure 8: Representative Western blot images in .jpg format of PEG10-Dendra2 levels transfected into PEG10 KO cells, and of Proteinase K protection assay. Adobe Illustrator file containing VLP size distribution graph.

Supp Figure 9: Representative Western blot images in .jpg format and and quantification (in .prism and .xlsx format) of a VLP isolation experiment carried out on cells treated with siG3BP1 and siG3BP2. The .zip file contains images of blots in .sgd and .jpg format from three independent replicates.

 

Code/software

Description: Cell Profiler pipelines and R code: Files containing Cell Profiler pipelines and R code used in various figures of the paper named accordingly.

Files with .R extension are R code that were run on R v4.4.1 using the RStudio IDE v 2024.04.2 Build 764 and files with .cpproj are CellProfiler pipelines were run on CellProfiler v4.2.6. Files with .lif extension were analyzed using LAS X v 3.3.0.16799. Graphs (files with .pzfx or .prism extension) were generated using GraphPad Prism 10.

Packages that were used to run proteomic analyses and generate volcano plots for Figure 1B were dplyr v1.1.4, ggplot2 v3.5.1, ggrepel v0.9.5, ggExtra v0.10.1, ggprism v1.0.5, patchwork v1.2.0, and openxlsx v4.2.6.1. Packages that were used to generate heatmaps for Figure 6C and S7C were openxlsx v4.2.6.1, dplyr v1.1.4, ComplexHeatmap v2.20.0, and colorRamp2 v0.1.0. Packages that were used to run proteomic analyses, generate volcano plots and preliminary Venn diagrams for Figure 6B and S7B were dplyr v1.1.4, ggplot2 v3.5.1, ggrepel v0.9.5, ggExtra v0.10.1, ggprism v1.0.5, patchwork v1.2.0, openxlsx v4.2.6.1, ggpubr v 0.6.0, and ggvenn v 0.1.10.

Access information

Other publicly accessible locations of the data:


